terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/el/ElTest.java

```diff
@@ -403,6 +403,272 @@ public void test06_Query_Display() {
     }
 
     @Test
+    public void test06_09_NestedJavaBean() {
+        driver.get(applicationContextUrl);
+        driver.findElement(By.id("EL")).click();
+        driver.findElement(By.id("06_09")).click();
+
+        inputFieldAccessor.overrideValue(By.id("criteria.name"), "yamada",
+                driver);
+        inputFieldAccessor.overrideValue(By.id("criteria.age"), "20", driver);
+        inputFieldAccessor.overrideValue(By.id("rememberCriteria"), "true",
+                driver);
+
+        driver.findElement(By.id("searchButton")).click();
+        driver.findElement(By.id("pagination")).findElement(By.linkText("2"))
+                .click();
+
+        // output 06_09 Test
+        assertThat(
+                driver.findElement(By.id("criteria.name"))
+                        .getAttribute("value"), is("yamada"));
+        assertThat(
+                driver.findElement(By.id("criteria.age")).getAttribute("value"),
+                is("20"));
+        assertThat(
+                driver.findElement(By.id("rememberCriteria")).getAttribute(
+                        "value"), is("true"));
+
+        // screen capture
+        screenCapture.save(driver);

```

Comment:
Screen capture is taken automatically at the finish of test.
Therefore this step is unnecessary.